Dockerfile

```
@@ -1,16 +1,15 @@
-# base node image
 FROM node:16-bullseye as base
 
 RUN apt-get update && apt-get install -y openssl
-
 RUN mkdir /app
 WORKDIR /app
 ENV NODE_ENV=production
-
 ADD . .
```

The only files required for a yarn install are the yarn.lock and package.json files. This will then give you the minimum amount of files required to do the yarn install. You'll need to re-add `ADD . .` somewhere after the `RUN yarm install --production` line. I would add it after the `COPY --from=base /app /app` in the prod image to reduce intermediate images from changing source code.

```suggestion
ADD yarn.lock package.json ./
```

---

Good idea, changed the Dockerfile accordingly. Build was successful.